src/core/Icon/style.ts

```diff
@@ -3,32 +3,40 @@ import styled from "@emotion/styled";
 import { SvgIcon, SvgIconProps } from "@material-ui/core";
 import { FC } from "react";
 import { getColors, getIconSizes, Props } from "../styles";
-import { IconSizes } from "./map";
+import { IconNameToSizes } from "./map";
```

Comment:
use generic `IconNameToSizes` to determine available sizes, based on `props.sdsIcon`